j2735-2024/src/test/java/j2735/RoadSafetyMessage/RoadSafetyMessage_BankAngleTest.java

```diff
@@ -0,0 +1,16 @@
+package j2735.RoadSafetyMessage;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+
+import org.junit.jupiter.api.Test;
+
+public class RoadSafetyMessage_BankAngleTest {

```

Comment:
Please rename this and other test classes to match the name of the class being tested.

remove the "RoadSafetyMessage_",  not needed  since it's already in the RoadSafetMessage package.

Reply:
Resolved